packages/@expo/cli/src/export/exportStaticAsync.ts

```diff
@@ -92,11 +93,25 @@ export async function getFilesToExportFromServerAsync(
     files = new Map(),
   }: {
     manifest: ExpoRouterRuntimeManifest;
+    serverManifest: ExpoRouterServerManifestV1;
```

Comment:
I think this is missing in `exportApiRoutesStandaloneAsync`. Or needs to be typed as `serverManifest?: ExpoRouterServerManifestV1;`